aemedge/blocks/hero/hero.js

```diff
@@ -61,17 +60,27 @@ async function replacePlaceholderText(elem) {
  * @param {Element} block The footer block element
  */
 export default async function decorate(block) {
-  const isMediaBlend = block.classList.contains('media-blend') || getMetadata('template') === 'article';
+  const isArticle = getMetadata('template') === 'article';
+  const isMediaBlend = block.classList.contains('media-blend') || isArticle;
```

Comment:
This can be` isArticle || block.classList.contains('media-blend');` to short circuit in first expression.

Reply:
fixed